tests/playwright/pages/wp-admin-page.js

```diff
@@ -166,4 +171,22 @@ module.exports = class WpAdminPage extends BasePage {
 	async editWithElementor() {
 		await this.page.getByRole( 'link', { name: ' Edit with Elementor' } ).click();
 	}
+
+	async setPermalinks( structure = 'plain' ) {
```

Comment:
Nice!

Reply:
@Barcov This function is not used anywhere. Do we need it?

Reply:
@louiswol94 I used it initially in this PR. We'll be using this in Taxonomy filter soon. It's also useful to tests that Hein has written in Mega menu. So I've left it in after refactoring I can remove it and add it in a separate PR at some point if you prefer...

Reply:
@louiswol94 removing form this PR. Refactoring has orphaned it. I'll add it in again on a later PR.